get-started/part2.md

```diff
@@ -8,76 +8,96 @@ description: Learn how to create a Docker image by writing a Dockerfile, and use
 
 ## Prerequisites
 
-- Work through setup and orientation in [Part 1](index.md).
+Work through the orientation and setup in [Part 1](index.md).
 
 ## Introduction
 
-Now that we've got our orchestrator of choice set up in our development environment thanks to Docker Desktop,
+Now that we've set up our development environment, thanks to Docker Desktop,
 we can begin to develop containerized applications. In general, the development workflow looks like this:
 
 1. Create and test individual containers for each component of your application by first creating Docker images.
-2. Assemble your containers and supporting infrastructure into a complete application, expressed either as a *Docker stack file* or in Kubernetes YAML.
-3. Test, share and deploy your complete containerized application. 
+2. Assemble your containers and supporting infrastructure into a complete application.
+3. Test, share, and deploy your complete containerized application.
 
 In this stage of the tutorial, let's focus on step 1 of this workflow: creating the images that our containers will be based on. Remember, a Docker image captures the private filesystem that our containerized processes will run in; we need to create an image that contains just what our application needs to run.
 
 > **Containerized development environments** are easier to set up than traditional development environments, once you learn how to build images as we'll discuss below. This is because a containerized development environment will isolate all the dependencies your app needs inside your Docker image; there's no need to install anything other than Docker on your development machine. In this way, you can easily develop applications for different stacks without changing anything on your development machine.
 
-## Setting Up
+## Set up
 
-1.  Clone an example project from GitHub (if you don't have git installed, see the [install instructions](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) first):
+Clone an example project from GitHub (if you don't have git installed, see the [install instructions](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) first):
```

Comment:
I wonder if we should offer a way without installing Git. Maybe someone does not know what Git is and then leaves our docs page, struggles there and never comes back. 😅 

I saw that we have html divs to show some kind of tabs. Maybe offer tabs
- Git
- Windows without Git
- Mac/Linux without Git

---
With "Git" the two following commands that are already here are fine.

```
git clone https://github.com/dockersamples/node-bulletin-board
cd node-bulletin-board/bulletin-board-app
```

---
With "Windows without Git" the following three commands work

```
curl.exe -LO https://github.com/dockersamples/node-bulletin-board/archive/master.zip
tar.exe xf master.zip
cd node-bulletin-board-master\bulletin-board-app
```

The `curl.exe` (instead of just `curl`) is used to make it work in PowerShell terminal as we don't know which terminal the user opens. The backslash in the `cd` command is used to make it work both in cmd.exe and powershell.exe terminal.
curl.exe and tar.exe are available in Windows 10 since 1809, but 1803 and older releases are hopefully no longer supported 🤷‍♀ 

---
With "Mac/Linux without Git" the following three commands work

```
curl -LO https://github.com/dockersamples/node-bulletin-board/archive/master.zip
unzip master.zip
cd node-bulletin-board-master/bulletin-board-app
```

A macOS user sees this popup the first time running the `git` command. So maybe it's also better to offer a way without git.

<img width="594" alt="macos-first-time-git" src="https://user-images.githubusercontent.com/207759/73823296-3d0dd680-47f8-11ea-90ab-c2aa3256339a.png">

Reply:
This is a great idea. I'll update accordingly.

Reply:
I've added the tabs. PTAL